### PR TITLE
Fix Row Block selected state

### PR DESCRIPTION
**Pull request by richtabor**
This PR fixes an issue where the selected state of the Row block was misaligned due to recent updates in Gutenberg. 

**Before:** 
before

**After:**
Screen Shot 2019-06-16 at 8 31 38 PM

### CODE REVIEW

File: src/blocks/row/styles/editor.scss [hunk 1]
```diff
@@ -83,11 +84,12 @@
...
-				margin-top: -12px;
+				margin-top: 0;
+				// margin-top: -12px;
 
-				&::before {
-					top: -1px;
-				}
+				// &::before {
+				// 	top: -1px;
+				// }
```
Comment (jrtashjian) [inline src/blocks/row/styles/editor.scss]: Let's remove this code instead of leaving it in commented.

File: src/blocks/row/styles/editor.scss [hunk 2]
```diff
@@ -83,11 +84,12 @@
 
 			> .editor-block-list__block-edit {
 				margin-bottom: 0;
-				margin-top: -12px;
+				margin-top: 0;
+				// margin-top: -12px;
```
Comment (jrtashjian) [inline src/blocks/row/styles/editor.scss]: Let's remove this code instead of leaving it in commented.

File: src/blocks/row/styles/editor.scss [hunk 3]
```diff
@@ -54,8 +54,9 @@
 
 		// Adjust the individual column block.
 		> [data-type="coblocks/column"] {
-			display: flex;
-			flex-direction: column;
+			margin-left: 0;
+			// display: flex;
+			// flex-direction: column;
```
Comment (jrtashjian) [inline src/blocks/row/styles/editor.scss]: Let's remove this code instead of leaving it in commented.

**Comment by richtabor** [comment]
@EvanHerman Mind taking over these requested changes?

**Pull request merged**